pkg/kubernetes/client/database/database.go

```diff
@@ -47,40 +46,6 @@ type DBClusterClient struct {
 //nolint:gochecknoglobals
 var addToScheme sync.Once
 
-// NewForConfig creates a new database cluster client based on config.
-func NewForConfig(c *rest.Config) (*DBClusterClient, error) {
-	config := *c
-	config.ContentConfig.GroupVersion = &everestv1alpha1.GroupVersion
-	config.APIPath = "/apis"
-	config.NegotiatedSerializer = scheme.Codecs.WithoutConversion()
-	config.UserAgent = rest.DefaultKubernetesUserAgent()
-
-	var err error
-	addToScheme.Do(func() {
-		err = everestv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
-		metav1.AddToGroupVersion(scheme.Scheme, everestv1alpha1.GroupVersion)
-	})
-
-	if err != nil {
-		return nil, err
-	}
-
-	client, err := rest.RESTClientFor(&config)
-	if err != nil {
-		return nil, err
-	}
-
-	return &DBClusterClient{restClient: client}, nil
-}
-
-// DBClusters returns a db cluster client.
-func (c *DBClusterClient) DBClusters(namespace string) DBClusterInterface { //nolint:ireturn
-	return &dbClusterClient{
-		restClient: c.restClient,
-		namespace:  namespace,
-	}
-}
-
```

Comment:
Is there a point in keeping anything else in this file? By removing this method the `dbClusterClient` struct also becomes unused. The `addToScheme` gets unused by deleting `NewForConfig`.  The interfaces `DBClusterClientInterface` `DBClusterInterface` also seems to be not used anywhere.